xla/backends/profiler/gpu/device_tracer_rocm.cc

```diff
@@ -212,19 +210,18 @@ class RocmTraceCollectorImpl : public profiler::RocmTraceCollector {
  private:
   std::atomic<int> num_callback_events_;
   std::atomic<int> num_activity_events_;
-  uint64_t start_walltime_ns_;
-  uint64_t start_gputime_ns_;
+  uint64_t start_walltime_ns_;  //\TODO: Use tsl::uint64
```

Comment:
I think using the uint64_t type should be preferred here - I think we should delete these comments and keep it as is. (Also, our linter internally wants this comment to be formatted like `// TODO(user): message`)

Reply:
@ddunl  Thanks for the feedback. We have another set of commits on top of this PR that we are waiting to push.
Is it ok if we accept this one and then clean up this comment in the next PR? This way we can get the next set of changes lined up for merge.

Thanks!